css/style.css

```diff
@@ -0,0 +1,269 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500&display=swap');
```

Comment:
It's best practice to add linters to your project to help you clean and bug free codes